README.md

```diff
@@ -8,6 +8,7 @@ Not all subsites of www.haskell.org are built from this repository.
 Some of the others are
 
 * www.haskell.org/cabal (built from [cabal-website](https://github.com/haskell/cabal-website))
+* www.haskell.org/platform (built from [haskell-platform](https://github.com/haskell/haskell-platform/tree/master/website))
```

Comment:
I guess https://www.haskell.org/ghcup/ is missing too... the repo is here: https://gitlab.haskell.org/haskell/ghcup-hs/-/tree/master/www

Reply:
Thanks! I've added it to this PR.